store/sqlstore/channel_store_shared_channels.go

```diff
@@ -56,8 +56,23 @@ func (s SqlChannelStore) GetSharedChannels(offset, limit int, opts store.SharedC
 		return nil, errors.New("cannot exclude home and remote shared channels")
 	}
 
+	safeConv := func(offset, limit int) (uint64, uint64, error) {
+		if offset < 0 {
+			return 0, 0, errors.New("offset must be postive integer")
+		}
+		if limit < 0 {
+			return 0, 0, errors.New("limit must be postive integer")
+		}
+		return uint64(offset), uint64(limit), nil
+	}
```

Comment:
I think some form of this would perfectly fit in the `utils` package. It might be useful in more than one place for sure.